.github/workflows/test.yml

```diff
@@ -95,3 +95,47 @@ jobs:
       - name: Build
         run: pnpm run build
         working-directory: web
+      # - name: Install Playwright
+      #   run: pnpm run e2e-install
+      #   working-directory: web
+      # - name: Run Playwright tests
+      #   run: pnpm run e2e
+      #   working-directory: web
+      #   env:
+      #     # This might depend on your test-runner/language binding
+      #     PLAYWRIGHT_TEST_BASE_URL: djmoxvh6xpra8.cloudfront.net/
+  playwright-tests:
```

Comment:
Now that #32 is implemented, the playwright tests depend on the preview environment being created. Can you shift this job into `preview.yml` and make it dependent on the `deploy` job completing?